imagefiles/dockcross

```diff
@@ -193,13 +193,24 @@ else
     HOST_PWD=$PWD
 fi
 
+# Mount Additional Volumes
+if [ -z "$SSH_DIR" ]; then
+    SSH_DIR="$HOME/.ssh"
+fi
+
+HOST_VOLUMES=
+if [ -e "$SSH_DIR" ]; then
+    HOST_VOLUMES+="-v $SSH_DIR:/home/$(id -un)/.ssh"
```

Comment:
Windows & OS X are probably special cases because docker runs within the context of a VM on both.

I personally use Linux-based distros 90% of time, so I'm probably not the best person to ask; however, if there is a SHARED_DIR that links the HOST and VM environments ... if `.ssh/` resided in there, I don't see why those files wouldn't be picked up.